doc/rtd/applications.rst

```diff
@@ -1557,6 +1561,304 @@ Command Line Arguments
     displays both basic and advanced documentation together
 
 
+.. _teca_tempest_tc_detect:
+
+teca_tempest_tc_detect
```

Comment:
we should note in the documentation somewhere that this application is GPU-enabled

Reply:
I added the following sentence: "The cyclone detecton algorithm is based on two TempestExtremes codes :cite:`tempestextremes`: DetectNodes and StitchNodes. **It can be executed in parallel on CPUs and GPUs.**
More details about the use of the application can ben found `here`_."

What do you think?